src/CSET/operators/filters.py

```diff
@@ -21,30 +21,30 @@
 
 
 def filter_cubes(
-    cubelist: iris.cube.CubeList, constraint: iris.Constraint, **kwargs
+    cube: iris.cube.Cube, constraint: iris.Constraint, **kwargs
```

Comment:
```suggestion
    cube: Union[iris.cube.Cube, iris.cube.CubeList], constraint: iris.Constraint, **kwargs
```
Show that it can take either a cube or a cube list in the function signature.